planner/core/planbuilder.go

```diff
@@ -1470,6 +1512,89 @@ func (b *planBuilder) buildExplain(explain *ast.ExplainStmt) (Plan, error) {
 	return p, nil
 }
 
+// colToColumnDef converts expression.Column `c` to ColumnDef:
+// if `c` refers to a column of the source table, we copy some of the column attributes from the referred column.
+// if `c` is an expression(for example, `count(*)`), the original table/column cannot be found.
+func (b *planBuilder) colToColumnDef(c *expression.Column, t *ast.TableName) (*ast.ColumnDef, error) {
```

Comment:
Below logic is from [https://dev.mysql.com/doc/refman/5.7/en/create-table-select.html](https://dev.mysql.com/doc/refman/5.7/en/create-table-select.html), and this function is only used by `create-select` statement? Please mention it in the comments.

Reply:
1. Yes, it is only used by `create-select`
2. OK, it will be added